dts: arm: nordic: Fix SRAM partitioning

nordicjm opened the pull request:
Uses the correct way to partition memory as per the linux binding, also fixes names which were not compliant with the zephyr devicetree guidelines

nordicjm committed:
dts: arm: nordic: Add ranges and address/size cells to SRAM nodes

Allows these nodes to be partitioned

Signed-off-by: Jamie McCrae <jamie.mccrae@nordicsemi.no>

nordicjm committed:
dts: vendor: nordic: nrf91: Fix SRAM partitioning

Uses the correct way to partition memory as per the linux binding,
also fixes names which were not compliant with the zephyr
devicetree guidelines

Signed-off-by: Jamie McCrae <jamie.mccrae@nordicsemi.no>

nordicjm committed:
dts: vendor: nordic: nrf5340: Fix SRAM partitioning

Uses the correct way to partition memory as per the linux binding,
also fixes names which were not compliant with the zephyr
devicetree guidelines

Signed-off-by: Jamie McCrae <jamie.mccrae@nordicsemi.no>

aescolar reviewed (dismissed):
+1 for the files I maintain

nordicjm committed:
tests: ipc: ipc_session: Correct name from sample to test

This wrongly displayed in CI as a sample failing, when it is a
test and not a sample, this changes the name to have test in it

Signed-off-by: Jamie McCrae <jamie.mccrae@nordicsemi.no>

aescolar reviewed (approved):
+1
